### ISSUE TITLE
twisted/web/soap.py borked

### ISSUE THREAD

**Issue by twisted-trac** [twisted/twisted#10470]
|<img alt="bcornett's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50">| bcornett reported|
|-|-|
|Trac ID|trac#615|
|Type|defect|
|Created|2004-05-15 01:52:40Z|

<details><summary>Searchable metadata</summary>

```
trac-id__615 615
type__defect defect
reporter__bcornett bcornett
priority__high high
milestone__ 
branch__ 
branch_author__ 
status__closed closed
resolution__fixed fixed
component__ 
keywords__ 
time__1084585960000000 1084585960000000
changetime__1084586587000000 1084586587000000
version__ 
owner__ 
cc__bcornett
```
</details>

**Comment by twisted-trac** [comment]
|<img alt="bcornett's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50">|bcornett commented|
|-|-|

```
#!html
<pre>
twisted.web.soap.py doesn't seem to be working
no twisted experience, but this seems to help

--- /usr/lib/python2.3/site-packages/twisted/web/soap.py        2004-05-05
08:37:08.000000000 -0500
+++ /tmp/soap.py        2004-05-14 14:54:17.959827768 -0500
@@ -71,7 +71,7 @@
  
         p, header, body, attrs = SOAPpy.parseSOAPRPC(data, 1, 1, 1)
  
-        methodName, args, kwargs, ns = p._name, p._aslist, p._asdict, p._ns
+        methodName, args, kwargs, ns = p._name, p._aslist(), p._asdict(), p._ns
  
         # deal with changes in SOAPpy 0.11
         if callable(args):
</pre>
```

**Comment by twisted-trac** [comment]
|[<img alt="itamarst's avatar" src="https://avatars.githubusercontent.com/u/3266662?s=50" width="50" height="50">](https://github.com/itamarst)|@itamarst commented|
|-|-|

```
#!html
<pre>
This is identical to some bug whose number I'm not gonna search for. ANyway,
should be fixed in 1.3.0.
</pre>
```

**Comment by twisted-trac** [comment]
|<img alt="Automation's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50">|Automation removed owner|
|-|-|
